core/src/test/resources/edu/wpi/grip/projects/testALL.grip

```diff
@@ -405,9 +405,6 @@
       <grip:Input step="34" socket="1">
         <value>COLOR_BGR2GRAY</value>
       </grip:Input>
-      <grip:Input step="34" socket="2">
-        <value>0.0</value>
-      </grip:Input>

```

Comment:
It was incompatible with the new version of the operation. I think the compatibility test was disabled for so long that it was never picked up